.env.example

```diff
@@ -1,6 +1,6 @@
 SHEET_ID=
 GOOGLE_SHEETS_CLIENT_EMAIL=
 GOOGLE_SHEETS_PRIVATE_KEY=
-CONTENTFUL_PREVIEW_ACCESS_TOKEN=
-CONTENTFUL_ACCESS_TOKEN=
-CONTENTFUL_SPACE_ID=
+NEXT_CONTENTFUL_PREVIEW_ACCESS_TOKEN=
+NEXT_CONTENTFUL_ACCESS_TOKEN=
+NEXT_CONTENTFUL_SPACE_ID=
```

Comment:
The `NEXT_` prefix is needed for the value to be accessible. Without this I'm seeing an Authorization error, because the value is not coming through.